lte/gateway/c/core/oai/tasks/nas/emm/TrackingAreaUpdate.c

```diff
@@ -682,11 +682,8 @@ static int emm_tracking_area_update_accept(nas_emm_tau_proc_t* const tau_proc) {
       emm_sap.u.emm_as.u.establish.emergency_number_list = NULL;
 
       emm_sap.u.emm_as.u.establish.eps_network_feature_support =
-          calloc(1, sizeof(eps_network_feature_support_t));
-      emm_sap.u.emm_as.u.establish.eps_network_feature_support->b1 =
-          _emm_data.conf.eps_network_feature_support[0];
-      emm_sap.u.emm_as.u.establish.eps_network_feature_support->b2 =
-          _emm_data.conf.eps_network_feature_support[1];
+          (eps_network_feature_support_t*) &_emm_data.conf
```

Comment:
could this lead to eps_network_feature_support ptr being freed later and then another place trying to access it? I assume we do a nullptr check if that's the case, just want to make sure this doesn't lead to crash

Reply:
Here we are not actually allocating memory, just sending address of parameter, eps_network_feature_support which is part of configuration structure.
While accessing we have null check.